plugins/linux/ifplugin/linuxcalls/link_linuxcalls.go

```diff
@@ -232,3 +232,45 @@ func (h *NetLinkHandler) AddVethInterfacePair(ifName, peerIfName string) error {
 	}
 	return nil
 }
+
+// AddVRFDevice configures new VRF network device.
+func (h *NetLinkHandler) AddVRFDevice(vrfDevName string, routingTable uint32) error {
+	attrs := netlink.NewLinkAttrs()
+	attrs.Name = vrfDevName
+	link := &netlink.Vrf{
+		LinkAttrs: attrs,
+		Table:     routingTable,
+	}
+	if err := netlink.LinkAdd(link); err != nil {
+		return errors.Wrapf(err, "LinkAdd %v", link)
```

Comment:
This error will be printed as:
`failed to add VRF device vrf-1 (rt: 1): LinkAdd &{{0 0 -1 vrf-1  0 0 0 0 <nil>  <nil> 0 <nil>  <nil> unknown 0 0 0 []} 1}: operation not supported`

and adding variable `link` with `%v` verb does not seem very helpful.. since we build the link/attrs it should be enough to mention `vrfDevName` (`attrs.Name`) and `routingTable` (`link.Table`).